src/Libraries/Microsoft.Extensions.Resilience/README.md

```diff
@@ -18,6 +18,22 @@ Or directly in the C# project file:
 </ItemGroup>
 ```
 
+## Usage Examples
+
+The services can be registered using the following method:
+
+```csharp
+public static IServiceCollection AddResilienceEnricher(this IServiceCollection services)
+```
+
+This will optionally consume the `IExceptionSummarizer` service if it has been registered and add that data to Polly's telemetry. It will also include `RequestMetadata` that can be set or retrieved with these extensions:
```

Comment:
I wonder if the README should have a pointer to the exception summarizer package as well as the Telemetry one. That said, it probably makes more sense to point to conceptual docs describing these, so perhaps it's fine to keep as is and once we have those conceptual docs we just add a pointer to those here.

Reply:
Yeah, concept doc links would be more helpful.